### PR TITLE
[main] - FE Picasso copy update

### PR DESCRIPTION
**Pull request by easteregg**
## Issue
https://app.clickup.com/t/31evhjc

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-31evhjc 5. Copywrite confirmation](https://app.clickup.com/t/31evhjc)

File: frontend/apps/picasso/components/Organisms/Bond/ClaimForm.tsx [hunk 1]
```diff
@@ -52,12 +52,15 @@ export const ClaimForm = () => {
...
       (msg) => {
         closeSnackbar(snackbarKey);
-        enqueueSnackbar("An error occurred while processing transaction", {
-          variant: "error",
-          isClosable: true,
-          persist: true,
-          description: msg,
-        });
+        enqueueSnackbar(
+          "An error occurred while processing the transaction. The process was canceled.",
```
Comment (saadjhk) [inline frontend/apps/picasso/components/Organisms/Bond/ClaimForm.tsx:56]: cancelled

File: frontend/apps/picasso/components/Organisms/Staking/BurnModal.tsx [hunk 2]
```diff
@@ -55,12 +54,15 @@ export const BurnModal: FC<{
...
           (errorMessage: string) => {
             closeSnackbar(snackbarKey);
-            enqueueSnackbar("An error occurred while processing transaction", {
-              variant: "error",
-              isClosable: true,
-              persist: true,
-              description: errorMessage,
-            });
+            enqueueSnackbar(
+              "An error occurred while processing the transaction. The process was canceled.",
```
Comment (saadjhk) [inline frontend/apps/picasso/components/Organisms/Staking/BurnModal.tsx:58]: cancelled

File: frontend/apps/picasso/components/Organisms/Staking/RenewModal.tsx [hunk 3]
```diff
@@ -84,12 +81,15 @@ export const RenewModal: FC<{
...
           (errorMessage: string) => {
             closeSnackbar(snackbarKey);
-            enqueueSnackbar("An error occurred while processing transaction", {
-              variant: "error",
-              isClosable: true,
-              persist: true,
-              description: errorMessage,
-            });
+            enqueueSnackbar(
+              "An error occurred while processing the transaction. The process was canceled.",
```
Comment (saadjhk) [inline frontend/apps/picasso/components/Organisms/Staking/RenewModal.tsx:85]: cancelled

File: frontend/apps/picasso/defi/polkadot/hooks/useTransfer.ts [hunk 4]
```diff
@@ -62,13 +62,16 @@ export const useTransfer = () => {
...
-            url: SUBSTRATE_NETWORKS.picasso.subscanUrl + txHash,
-          });
+          snackbarKey = enqueueSnackbar(
+            "Executing transfer... just one moment, please.",
+            {
+              persist: true,
+              description: `Transaction hash: ${txHash}`,
+              variant: "info",
+              isCloseable: true,
+              url: SUBSTRATE_NETWORKS.picasso.subscanUrl + txHash,
```
Comment (saadjhk) [inline frontend/apps/picasso/defi/polkadot/hooks/useTransfer.ts:72]: sir this might need a rebase

**Review by saadjhk — CHANGES_REQUESTED** [review]

File: frontend/apps/picasso/components/Organisms/Bond/ClaimForm.tsx (hunk 1 above)
Comment (easteregg) [reply frontend/apps/picasso/components/Organisms/Bond/ClaimForm.tsx:56]: https://www.merriam-webster.com/words-at-play/canceled-or-cancelled

File: frontend/apps/picasso/defi/polkadot/hooks/useTransfer.ts (hunk 4 above)
Comment (easteregg) [reply frontend/apps/picasso/defi/polkadot/hooks/useTransfer.ts:72]: Rebase done.

**Review by saadjhk — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Pull request merged**